docs/platform/integrations/metrics/prometheus.rst

```diff
@@ -0,0 +1,159 @@
+Using Aiven with Prometheus
+===========================
+
+.. _h_6a84674413:
+
+Background
+----------
+
+Prometheus is an open-source systems monitoring and alerting toolkit. It
+implements in-memory and persistent storage model for metrics as well as
+a query language for accessing the metrics. Prometheus is the third
+metrics integration supported by Aiven after the `Aiven InfluxDB &
+Grafana
+integration <https://help.aiven.io/services/integrations/getting-started-with-service-integrations>`__
+and `Datadog
+integration <https://help.aiven.io/services/integrations/getting-started-with-datadog>`__
+.
+
+The metrics delivery model of Prometheus is a somewhat atypical pull
+model where the Prometheus server connects to HTTP servers running on
+the nodes being monitored and pulls the metrics from them. While this
+makes the service discovery more of an issue than with the more common
+push approach, it does have the benefit of making the metrics available
+not just for Prometheus but for any app that can read the Prometheus
+format from the HTTP server running on Aiven nodes.
+
+Enabling Prometheus support in Aiven
+------------------------------------
+
+To enable the metrics endpoints that Prometheus can query you need to
+follow the steps below:
+
+#. | Navigate to the project ``Service Integrations`` section from the
+     side menu
+
+   .. image:: /images/platform/integrations/prometheus-side-menu.png
```

Comment:
Are there any preferred naming convention or standardisation for the alt text?

Reply:
It's difficult to standardise on alt text, but it should describe what's in the image so that if someone can't see the image, they get enough information that they can still understand the article